src/main/java/io/confluent/kafkarest/SimpleConsumerFactory.java

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package io.confluent.kafkarest;
+
+import kafka.consumer.ConsumerConfig;
+import kafka.javaapi.consumer.SimpleConsumer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Properties;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class SimpleConsumerFactory {
+
+  private static final Logger log = LoggerFactory.getLogger(SimpleConsumerManager.class);
+
+  private final KafkaRestConfig config;
+
+  private final ConsumerConfig consumerConfig;
+  private final AtomicInteger clientIdCounter;
+
+  public SimpleConsumerFactory(final KafkaRestConfig config) {
+    this.config = config;
+
+    clientIdCounter = new AtomicInteger(0);
+
+    final Properties props = (Properties) config.getOriginalProperties().clone();
+    // ConsumerConfig is intended to be used with the HighLevelConsumer. Therefore, it requires some properties
+    // to be instantiated that are useless for a SimpleConsumer.
+    // We use ConsumerConfig for our SimpleConsumer, because it contains sensible defaults (buffer size, ...).
+    // Maybe, it would make more sense to directly define these defaults here or in a SimpleConsumerConfig ?
```

Comment:
Yeah, I feel like this is a bit confusing since the code here doesn't indicate what values are being used. Even though it'll just be a class of constants for now, I think it makes more sense to separate this out into a SimpleConsumerConfig and declare fields for the settings you'll actually use and just use the ConsumerConfig to initialize them, with the explanation about sensible defaults given here.